tests/e2e/utils/click-on-more-menu-item.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * External dependencies
+ */
+import { first } from 'lodash';
+
+/**
+ * Clicks on More Tools & options menu item by label.
```

Comment:
This could also use a comment that it's almost a copy from upstream but uses the more reliable way for clicking.